CONTRIBUTING.md

```diff
@@ -0,0 +1,189 @@
+# Contributing
```

Comment:
All of these docs are not relevant to consumers of the package, only to contributors.